app/styles/new-signup.css

```diff
@@ -0,0 +1,88 @@
+.container {
+    width: 100%;
+    color: #666;
+    text-align: center;
+    padding: 1em 0;
+}
+
+.user-details {
+    display: flex;
+    padding: 2rem;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    max-width: 80%;
+    margin: auto;
+    background-color: #f0f2f5;
+}
+
+.user-details>* {
```

Comment:
Can we please put the spacing between the classname, child selector, and universal selector ?

Reply:
okay will do that